src/components/section-navigation/_macro.njk

```diff
@@ -15,8 +16,12 @@
                                 {% set isCurrent = false %}
                             {% endif %}
                             <li class="ons-section-nav__item{% if item.classes %} ' ' + {{ item.classes }}{% endif %}{% if isCurrent == true %} ons-section-nav__item--active{% endif %}">
-                                <a class="ons-section-nav__link" href="{{ item.url }}"{% if isCurrent == true %} aria-current="location"{% endif %}>{{ item.title }}</a>
-                                {% if item.anchors and isCurrent == true %}
+                            {% if isCurrent == true %} 
+                                <h{{headingLevel + 2 if section.title else headingLevel + 1}} class="ons-section-nav__link ons-section-nav__item-header" href="{{ item.url }}" aria-current="location">{{ item.title }}</h{{headingLevel + 2 if section.title else headingLevel + 1}}>
+                            {% else %}
+                                <a class="ons-section-nav__link" href="{{ item.url }}">{{ item.title }}</a>
+                            {% endif %}
+                            {% if item.anchors and isCurrent == true %}
```

Comment:
There is duplication of logic here, could we not move this to the top and use a param like the `headingLevel` one